regparser/commands/pipeline.py

```diff
@@ -31,13 +36,17 @@ def pipeline(ctx, cfr_title, cfr_part, output, only_latest, xml_ttl):
     * a directory prefixed with "git://". This will export to a git
       repository"""
     params = {'cfr_title': cfr_title, 'cfr_part': cfr_part}
-    ctx.invoke(sync_xml, xml_ttl=xml_ttl)
-    if only_latest:
-        ctx.invoke(current_version, **params)
-    else:
-        ctx.invoke(versions, **params)
-        ctx.invoke(annual_editions, **params)
-        ctx.invoke(fill_with_rules, **params)
-    ctx.invoke(layers, **params)
-    ctx.invoke(diffs, **params)
-    ctx.invoke(write_to, output=output, **params)
+    try:
+        ctx.invoke(sync_xml, xml_ttl=xml_ttl)
+        if only_latest:
+            ctx.invoke(current_version, **params)
+        else:
+            ctx.invoke(versions, **params)
+            ctx.invoke(annual_editions, **params)
+            ctx.invoke(fill_with_rules, **params)
+        ctx.invoke(layers, **params)
+        ctx.invoke(diffs, **params)
+        ctx.invoke(write_to, output=output, **params)
+    except pyparsing.ParseException as exc:
```

Comment:
We spoke about this briefly, so documenting here:
* ParseException is special because the exception message doesn't include the text it was attempting to parse
* Putting this inside `eregs.py` will allow the exception to be caught (and a more useful debug message be displayed) for all subcommands